c++/Nbody.cpp

```diff
@@ -0,0 +1,166 @@
+#pragma once
+#include <iostream>
+#include <cmath>
+#include <vector>
+#include <iostream>
+#include <vector>
+#include <numeric>
+#include <cmath>
+#include <chrono>
+
+
+using namespace std;
+
+ const  double G = 0.6;//6.67428e-11;
+ const  double AU = 149.6e6 * 1000;
+ const double scale = 1; // 1.5e-22;
+ const double timestep = 0.01; // (3600 * 24 * 365 * 2000000000) * 2000000000;//(3600 * 24 * 365 * 2090000000) * 100000000;
+ const int b_bass_mass = 4000000;
```

Comment:
Should these constants not be in screaming snake case?

Reply:
Probably

Reply:
Yes, they should